code/obj/item/misc_junk.dm

```diff
@@ -642,3 +642,25 @@ TYPEINFO(/obj/item/reagent_containers/vape)
     The danger is unleashed only if you substantially disturb this place physically. This place is best shunned and left uninhabited.<br>
 	<br>
 	...spooky!"}
+
+/obj/item/boarvessel
+	name = "Boar Vessel, 600-500 BC, Etruscan, ceramic"
+	desc = "Oh my God! A REAL Boar Vessel, 600-500 BC, Etruscan, ceramic."
+	icon_state = "boarvessel"
+
+	attack_hand(mob/user)
+		if (user.a_intent == "help")
```

Comment:
Note that this prevents picking it up on help intent. Perhaps only do this when in hand?

Reply:
I'm not sure how to do this?

Reply:
Gnome chompsky has this behaviour if you are stumped!

Reply:
Ah! Thanks @DimWhat

Reply:
Fixed issue in commit [216333e](https://github.com/goonstation/goonstation/pull/14545/commits/216333ea6f5364a08aeee14e087ac736408c6a39)